alex-sofroniev/Homework1/CV/Alex's Curiculum.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html>
+<head>
+<title>Alex's Sofroniev - Curriculum</title>
+
+<meta name="viewport" content="width=device-width"/>
+<meta name="description" content="The Curriculum Vitae of Alex Sofroniev."/>
+<meta charset="UTF-8"> 
+
+<link type="text/css" rel="stylesheet" href="style.css">
+<link href='http://fonts.googleapis.com/css?family=Rokkitt:400,700|Lato:400,300' rel='stylesheet' type='text/css'>
+
+ 
+
+</head>
+<body id="top">
+<div id="cv" class="instaFade">
+	<div class="mainDetails">
+		<div id="headshot" class="quickFade">
+			<img src="myself.jpg" alt="Alex Sofroniev" />
+		</div>
+		
+		<div id="name">
+			<h1 class="quickFade delayTwo">Alex Sofroniev</h1>
+			<h2 class="quickFade delayThree">Programmer at PHP Masters</h2>
+		</div>
+		
+		<div id="contactDetails" class="quickFade delayFour">
+			<ul>
+				<li>email: <a href="mailto: sofroniev55@gmail.com" target="_blank">sofroniev55@gmail.com</a></li>
+				<li>website: <a href="file:///C:/Users/Allex%20Sofroniev/Desktop/php%20masters/lesson%205/blog.html" target="_blank">PHP Masters Blog</a></li>
+				<li>mobile: 0878274422</li>
+				<li>Date of birth: 02.10.2001</li>
+			</ul>
+		</div>
+		<div class="clear"></div>
+	</div>
+	
+	<form action="file:///C:/Users/Allex%20Sofroniev/Desktop/php%20masters/homeworks/homework%20lesson3/Autobiography-homework.html"> 
+
+	<div id="mainArea" class="quickFade delayFive">
+		<section>
+			<article>
+				<div class="sectionTitle">
+					<h1>Personal Profile</h1>
+				</div>
+			
+					<div class="sectionContent">
+						<p>I am Alex Sofroniev a 14 years old programmer at PHP Master's.I like to play football, train tennis, do downhills with my mountain bike, listen to music, make music, watch tv, play with my little brother, study technologies and programming, going out with my friends, do parkour, going out with my girlfriend, train in the fitness, swim in the pool, being healthy and eating healthy, go to shopping centers, eat ice cream and go to the mountain for a walk.</p>
+					</div>
+				</article>
+			<div class="clear"></div>
+		</section>
+		
+		
+		<section>
+			<div class="sectionTitle">
+				<h1>Professional Experience</h1>
+			</div>
+			
+			<div class="sectionContent">
+				<article>
+					<h2>PHP Master's</h2>
+					<p class="subDetails">Мay.09.2016 - Until Now</p>
+						<ul>
+							<li>HTML&CSS Developer</li>
+						</ul>
+						<br>
+				</article>
+
+<hr>
+				<article>
+	
+						
+							<h2>Biodit</h2>
+							<p class="subDetails">October 2015 - Until Now</p>
+
+								
+									<ul class="Biodit">
```

Comment:
@alex5505 
For this and all other occurrences:

It is bad idea to use unordered lists in such a way. You have created many lists with single list items. You should create single list with many list items ^_^ for example:

<ul>
    <li> Product Manager </li>
    <li> Product Developer </li>
    <li> Product Overview </li>
</ul>